examples/burgers1d.ipynb

```diff
@@ -118,175 +117,78 @@
     "\n",
     "autoencoder.load_state_dict(autoencoder_param)\n",
     "\n",
-    "# Z = autoencoder.encoder(X_train).detach().numpy()\n",
-    "Z = autoencoder.encoder(X_train)\n",
-    "Z0 = initial_condition_latent(param_grid, physics, autoencoder)\n",
-    "Zis_samples, gp_dictionnary, interpolation_data, sindy_coef, n_coef, coef_samples = simulate_interpolated_sindy(param_grid, Z0, t_grid, n_samples, Dt, Z, param_train, fd_type)\n",
-    "Zis_mean, _, _, _, _, _ = simulate_interpolated_sindy_mean(param_grid, Z0, t_grid, Dt, Z, param_train, fd_type)\n",
+    "n_coef = sindy.ncoefs\n",
+    "\n",
+    "from lasdi.gplasdi import sample_roms, average_rom\n",
+    "Zis_samples = sample_roms(autoencoder, physics, sindy, gp_dictionnary, param_grid, n_samples)\n",
+    "Zis_mean = average_rom(autoencoder, physics, sindy, gp_dictionnary, param_grid)\n",
+    "\n",
+    "print(Zis_mean.shape)\n",
+    "print(Zis_samples.shape)\n",
+    "\n",
+    "X_pred_mean = autoencoder.decoder(torch.Tensor(Zis_mean)).detach().numpy()\n",
+    "X_pred_samples = autoencoder.decoder(torch.Tensor(Zis_samples)).detach().numpy()\n",
+    "\n",
+    "avg_rel_error = np.zeros(param_grid.shape[0])\n",
+    "for k in range(param_grid.shape[0]):\n",
+    "    avg_rel_error[k], _ = compute_errors(X_pred_mean[k], physics, X_test[k].numpy())\n",
```

Comment:
I think `X_test` is already a numpy array (we never feed it through the autoencoder), so calling `X_test[k].numpy()` throws an error

Reply:
In the current framework `X_test` is now saved as `torch.Tensor`. Running the example with the up-to-date data should work in this notebook.